tests/components/airzone/test_sensor.py

```diff
@@ -34,50 +34,50 @@ async def test_airzone_create_sensors(
     assert state.state == "43"
 
     # WebServer
-    state = hass.states.get("sensor.webserver_rssi")
+    state = hass.states.get("sensor.mock_title_rssi")
```

Comment:
> It's ye boy unit tests back at it again with a strange situation.
> 
> I would expected that if I add
> 
> ```
> AZD_FULL_NAME: "something"
> ```
> 
> in the `HVAC_WEBSERVER_MOCK` in util.py, that it would fill the name. In fact, it does not. And to make it even stranger, adding that line to that object, even messed up the snapshot since it for some reason doesn't want the wifi-ssid in there anymore. Any ideas?

In that case, the aioairzone library is just returning the same for both AZD_MODEL and AZD_FULL_NAME, which is the device type:
https://github.com/Noltari/aioairzone/blob/fd567e3ea58dc9491f52995c9a0dc2b1efebcfae/aioairzone/webserver.py#L111C11-L113
https://github.com/Noltari/aioairzone/blob/fd567e3ea58dc9491f52995c9a0dc2b1efebcfae/aioairzone/common.py#L223-L230
<img width="351" alt="image" src="https://github.com/home-assistant/core/assets/264346/adeb6c7d-86c3-40b5-a96a-08cbadae5745">

Reply:
So what change should happen for the mock to have a device name?

Reply:
`API_WS_TYPE: API_WS_AZ,`
https://github.com/Noltari/home-assistant-core/commit/35714e0c50bd68b397747188a97d7bfa8772b43e